assets/scss/6-components/blast-header/blast-header.html

```diff
@@ -0,0 +1,11 @@
+<header class="c-blast-header l-align-center-children has-bg-black-off has-text-white has-l-btm-marg" style="background-image: url('/img/blast-header/TT-Blast-landing_photo.jpg')">
```

Comment:
Is the thought on the inline style that the image location will vary by app and thus doesn't make sense to include at the class level?

Reply:
yep exactly